notebooks/WET/WET-013_exposure-time.ipynb

```diff
@@ -0,0 +1,439 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    data_ids = list(butler.registry.queryDataIds(('exposure', 'visit', 'detector'), collections=collection_name, datasets='zernikeEstimateAvg'))
</pre><p>If possible, could we update it to use zernikes astropy table? Since at some point we will deprecate Avg and Raw?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/ts_aos_analysis/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Updated